src/System.Drawing.Common/tests/Performance/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
Should we run the perf tests on netcoreapp only? The netstandard implementation is a turd assembly. In fact the regular tests only have a netcoreapp and desktop configurations

Reply:
Do we run them anywhere else? I was just copy/pasting from FileSystem. If we can/should specify netcoreapp I'm happy to change it.

Reply:
I’m don’t think we yet do but if in the future we start it might cause pain to whoever brings the infra to do so, but I can live with this being NS